src/modules/commander/state_machine_helper.cpp

```diff
@@ -531,6 +538,18 @@ bool set_nav_state(vehicle_status_s *status, actuator_armed_s *armed, commander_
 		}
 		break;
 
+	case commander_state_s::MAIN_STATE_CUSTOM: {
```

Comment:
At a minimum should be handled like position control mode (could even be a fall through).